dumpling/export/writer_test.go

```diff
@@ -36,7 +36,29 @@ func TestWriteDatabaseMeta(t *testing.T) {
 
 	bytes, err := ioutil.ReadFile(p)
 	require.NoError(t, err)
-	require.Equal(t, "/*!40101 SET NAMES binary*/;\nCREATE DATABASE `test`;\n", string(bytes))
+	require.Equal(t, "/*!40101 SET NAMES binary*/;\n/*T![placement] SET PLACEMENT_CHECKS = 0*/;\nCREATE DATABASE `test`;\n", string(bytes))
+}
+
+func TestWritePolicyMeta(t *testing.T) {
+	t.Parallel()
+
+	dir := t.TempDir()
+	config := defaultConfigForTest(t)
+	config.OutputDirPath = dir
+
+	writer, clean := createTestWriter(config, t)
+	defer clean()
+
+	err := writer.WritePolicyMeta("test", "create placement policy `y` followers=2")
```

Comment:
if there are two placement policies `x` and `y`, they will all be written to the same `placement-policy-create.sql` file? 🤔

Reply:
yes, because placement policies are independent and have global scope, they can be applied on the multi databases/tables/partitions, create sql are like 
```sql
create placement policy `x` ...;
create placement policy `y` ...;
```

Reply:
@sylzd imo these should be placed into separate files i.e. `x-policy-create.sql` contains ``create [or replace] placement policy `x` ...;`` and `y-policy-create.sql` contains ``create placement policy `y` ...;`` etc.

Reply:
why should we make it separated? I think it's different from databases, it looks more like some initial sources and schemas or tables can use it for need, Do we need to dumpling only one policy like args `-B` or `-T`?

Reply:
fix it for unified specification~